merge/merger.go

```diff
@@ -30,8 +31,77 @@ func NewMerger(fhirHost string) *Merger {
 // is successful a new FHIR Bundle containing the merged patient record is returned.
 // If a merge fails, a FHIR Bundle containing one or more OperationOutcomes is
 // returned detailing the merge conflicts.
-func (m *Merger) Merge(source1, source2 string) (mergeID string, outcome *models.Bundle, err error) {
-	return mockMerge(source1, source2)
+func (m *Merger) Merge(source1ID, source2ID string) (mergeID string, outcome *models.Bundle, err error) {
+
+	var isMatched bool
+
+	source1 := new(models.Bundle)
+	source2 := new(models.Bundle)
+
+	//Unmarshal JSON to source objects
+	populateBundle(m.fhirHost+"/Bundle/"+source1ID, source1)
```

Comment:
Perhaps this could be something more like the following:

```go
bundle1, err := getBundle(source1)
```

That way we can explicitly check the error from this operation. It also seems a little more compact. What do you think?